.errcheck.exclude

```diff
@@ -1,2 +1,4 @@
 io.Copy
 (net.Conn).Close
+fmt.Fprintf
```

Comment:
I'm assuming this is new, do you have an example handy of the errors it is masking?

Reply:
@wrboyce these are the things that errcheck caught

```
resin/main.go:116:15:	fmt.Fprintln	fmt.Fprintln(os.Stderr, "Error: Resin API Key is required.")
resin/main.go:123:15:	fmt.Fprintln	fmt.Fprintln(os.Stderr, "Error: dir must be absolute.")
resin/main.go:133:15:	fmt.Fprintf	fmt.Fprintf(os.Stderr, "%s: No such file or directory\n", viper.Get(key))
resin/main.go:159:15:	fmt.Fprintf	fmt.Fprintf(os.Stderr, "%s", err)
```
and this ignore I added because it seems that's one of the example as well for errcheck: https://github.com/kisielk/errcheck#the-deprecated-method

But if there's a better way (e.g. fixing up the code instead of adding more ignores), that I'm open too!